.github/pull_request_template.md

```diff
@@ -1,23 +1,30 @@
+# Pull Requests
+
+A Pull Request should close a **single** issue; multiple issues can be closed when the issues are **small and related**, but that should be an exception not the rule. Please
+keep Pull Requests small and targeted; large 'code drops' with dozens of files will be closed and asked to split into reviewable pieces. Reviews that need to be large due to dependencies will be
+reviewed on a case-by-case basis.
+
 ## Related Issue
+
 Please use one of the well-known [github fixes keywords](https://help.github.com/en/articles/closing-issues-using-keywords) to reference
-the issues fixed with this PR (eg Fixes #<github issue number>). If an issue doesn't yet exist please create one if reasonable to aid 
+the issue fixed with this PR (eg Fixes #<github issue number>). If an issue doesn't yet exist please create one to aid
 in issue tracking.
 
 **NOTE**: For multiple issues resolved by this PR use the corresponding keywords **every time** in a comma-delimited list per the reference
 page above.
 
 ## Description
-Create a [descriptive title and provide all of the relevant details in the description](https://chris.beams.io/posts/git-commit/). 
-This information lives with the code and will help future readers (***including yourself***) and will serve as documentation.
 
-At the very least please describe the issue you are addressing, and what your fix entails. 
+For non-trivial bug fixes or for new features please describe how the issue was fixed or how the feature was implemented, from a summary level. This information will be used to help provide context to the reviewers of the pull request and should be additive to the issue being closed.
```

Comment:
```suggestion
Please describe how the issue was fixed or how the feature was implemented, from a summary level. This information will be used to help provide context to the reviewers of the pull request and should be additive to the issue being closed.
```

Description would be nice for trivial fixes too :)

Reply:
Agreed

Reply:
For trivial fixes, the description is the code :)

Reply:
"Trivial" is in the eye of the beholder :)

I agree description is good for all fixes.